code/go/0chain.net/blobbercore/stats/blobberstats.go

```diff
@@ -292,6 +352,16 @@ func (bs *BlobberStats) loadAllocationStats(ctx context.Context) {
 			zap.Error(err))
 		return
 	}
+
+	var count int64
+	err = db.Table("reference_objects").Distinct("allocation_id").Count(&count).Error
```

Comment:
`distinct` is low performance. please use `where deleted_at is null` to filter deleted file

Reply:
done